x/incentive/errors.go

```diff
@@ -7,19 +7,17 @@ import (
 )
 
 var (
-	// 0 = TODO
-	ErrNotImplemented = errors.Register(ModuleName, 0, "not implemented")
-
 	// 1XX = General
-	ErrInvalidProgramID     = errors.Register(ModuleName, 100, "invalid program ID")
-	ErrNilAsset             = errors.Register(ModuleName, 101, "nil asset")
-	ErrInvalidTier          = errors.Register(ModuleName, 102, "invalid unbonding tier")
-	ErrEmptyAddress         = errors.Register(ModuleName, 103, "empty address")
-	ErrInvalidProgramStatus = errors.Register(ModuleName, 104, "invalid program status")
-
-	// 2XX = Params
-	ErrUnbondingTierOrder   = errors.Register(ModuleName, 200, "unbonding tier lock durations out of order")
-	ErrUnbondingWeightOrder = errors.Register(ModuleName, 201, "unbonding tier weights out of order")
+	ErrInvalidProgramID        = errors.Register(ModuleName, 100, "invalid program ID")
```

Comment:
Can we use standard `ErrInvalidRequest` instead of creating new error types? These errors are already deprecated.

Reply:
Keeping as-is for now, but we can remove custom errors from this and all other modules at some later date if deemed necessary.